nucypher/blockchain/eth/agents.py

```diff
@@ -213,7 +229,8 @@ def get_locked_tokens(self, staker_address: str, periods: int = 0) -> int:
         return self.contract.functions.getLockedTokens(staker_address, periods).call()
 
     def owned_tokens(self, staker_address: str) -> int:
-        return self.contract.functions.getAllTokens(staker_address).call()
+        staker_info = self.get_staker_info(staker_address)
+        return staker_info[0]
```

Comment:
just my opinion: calling `getAllTokens` is more explicit than using `staker_info`

Reply:
✅